includes/reports/data/charts/v2/class-manifest.php

```diff
@@ -449,34 +480,22 @@ public function get_chart_options() {
 					'display' => $this->get_endpoint()->get_label() && $this->get_endpoint()->get( 'show_chart_title' ),
 					'text'    => $this->get_endpoint()->get_label(),
 				),
-				'scales'     => array(
-					'xAxes' => array(
-						array(
-							'type'     => 'time',
-							'display'  => true,
-							'ticks'    => array(
-								'source'      => 'auto',
-								'maxRotation' => 0,
-							),
-							'position' => 'bottom',
-							'time'     => array(
-								'unit'          => $time_unit,
-								'tooltipFormat' => $time_format,
-							),
-						),
-					),
-					'yAxes' => array(
-						array(
-							'type'     => 'linear',
-							'display'  => true,
-							'position' => 'left',
-						),
-					),
+				'scales'    => array(
+					'xAxes' => array(),
+					'yAxes' => array(),
 				),
 			);
+
+			// Check if specific axes are missing from the endpoint options and load them from defaults.
```

Comment:
Wondering if the `$default_xAxes` and `$default_yAxes` definitions could be moved down here as this is where they're used. Or actually move them to a private method to be called if needed?